leverage/container.py

```diff
@@ -174,7 +175,20 @@ def ensure_image(self):
 
         logger.info("Required Docker image not found.")
 
-        stream = self.client.api.pull(repository=self.image, tag=self.image_tag, stream=True, decode=True)
+        try:
+            stream = self.client.api.pull(repository=self.image, tag=self.image_tag, stream=True, decode=True)
+        except NotFound as e:
+            logger.error(f"The specified toolbox version, '{self.image_tag}' (toolbox image '{self.image}:{self.image_tag}') can not be found. If you come from a project created with an older version of Leverage CLI or have modified the version in build.env manually, please consider doing one of the following steps: delete the build.env file or set in this file the default version which is '{__toolbox_version__}'. (i.e. 'TERRAFORM_IMAGE_TAG={__toolbox_version__}')")
```

Comment:
Maybe we should break the message into multiple lines to give it a little more readability.
```suggestion
            logger.error(f"The specified toolbox version, '{self.image_tag}' (toolbox image '{self.image}:{self.image_tag}') can not be found. "
                         "If you come from a project created with an older version of Leverage CLI or have modified the 'build.env' file manually, "
                         f"please consider either deleting the file, or configuring a valid toolbox version to use. (i.e. 'TERRAFORM_IMAGE_TAG={__toolbox_version__}')")
```
Also, what do you think of this slightly shorter version?